Meadowlark-js/docker/readme.md

```diff
@@ -7,17 +7,8 @@ These compose files require [Docker Compose v2](https://github.com/docker/compos
 (which comes with Docker Desktop for Windows users). They provision the following
 services, all using local volumes to for permanent data storage:
 
-* [MongoDB](mongo/): starts a 3-node replica set of MongoDB 4.0 instances on ports 27017, 27018, and 27019.
-* [OpenSearch](opensearch/): single node cluster of OpenSearch database and [OpenSearch
-  Dashboard](http://localhost:5601/) (latest versions).
 * [PostgreSQL](postgres/): single instance of PostgreSQL 14.
```

Comment:
@stevenarnold-ed-fi Please note above how Brad has moved docker files into the appropriate backend. Let's do this for postgresql in the _next_ PG ticket. Waiting won't hurt and avoids potential rebase ugliness.